commands/new_function.go

```diff
@@ -74,16 +103,6 @@ func runNewFunction(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	if len(args) < 1 {
```

Comment:
What happened to these lines of code?

Reply:
The validations are moved to [preRunNewFunction()](https://github.com/openfaas/faas-cli/pull/375/files/8df892ee565df83faac1726f7c0b2b793b82f229#diff-4e0fbf5ad1c657e2d3f8a554849f6f25R68)